src/navigation/MainTabNavigator.tsx

```diff
@@ -48,7 +52,8 @@ const MainTabNavigator: FunctionComponent = () => {
   const HomeIcon: FunctionComponent<TabBarIconProps> = ({ focused, size }) => {
     return (
       <SvgXml
-        xml={focused ? Icons.HomeActive : Icons.HomeInactive}
+        xml={TabBarIcons.House}
+        fill={determineIconFill(focused)}
```

Comment:
Love taking advantage of the fill prop here 🙌  I don't think this would be too bad to inline:
`fill={focused ? Colors.primaryViolet : Colors.quaternaryViolet}` and might have the slight benefit of keeping the decision making logic close to the render logic, but this is 100% a style preference and your call!

Reply:
Yeah I actually went back and forth between these two options and went with wrapping it in a function to avoid repeating the colors. I could go either way though!